lib/meta-feed-helpers.js

```diff
@@ -184,7 +185,9 @@ module.exports = (ssb) => {
           toPullStream()
         ),
         // find only init for epoch zero
-        pull.filter(rootMsg => rootMsg.value.content?.tangles?.group?.root === null),
+        pull.filter(
+          (rootMsg) => rootMsg.value.content?.tangles?.group?.root === null
```

Comment:
with a specific `isInitZero` or `isInitRoot` we could replace this line with a very tight check.
I would be keen to see that because just checking the type and tangle isn't safe enough to lean on it for other fields.

Reply:
idk we don't really use the other fields here, we just want the msgKey I think and the schema doesn't help verify that

Reply:
ok. the other thing we could do (maybe make it a TODO) is the put this filter in the query.
i.e. that it is a group tangle root. It would be much faster to not have to parse the json of each `group/init` before checking... but this is just an optimisation